ui/src/reusable_ui/components/dropdowns/DropdownDivider.tsx

```diff
@@ -0,0 +1,16 @@
+import React, {SFC} from 'react'
+import classnames from 'classnames'
+
+interface Props {
+  text?: string
+}
+
+const DropdownDivider: SFC<Props> = ({text}) => (
+  <div className={classnames('dropdown--divider', {line: !text})}>{text}</div>
+)
+
+DropdownDivider.defaultProps = {
```

Comment:
what do you think about type-annotating this as a `Partial<Props>`?

Reply:
I don't know how to do it, investigated a little and couldn't find an example of doing it this way